uniffi_bindgen/src/scaffolding/templates/ErrorTemplate.rs

```diff
@@ -1,20 +1,48 @@
 {#
-    // For each error declared in the UDL, using the [Error] attribute, we assume the caller has provided a corresponding
-    // rust Error enum with the same name. We provide the traits for sending it across the FFI, which will fail to
-    // compile if the provided enum has a different shape to the one declared in the UDL. 
-    // Here we define the neccessary converstion to allow the error to propegate through the FFI as an error.
+// For each enum declared in the UDL, we assume the caller has provided a corresponding
```

Comment:
IIUC, I think you still mean "For each error" rather than "For each enum", since the close link between errors and enums is not necessarily that obvious in the UDL.